atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/output/OutputPrinter.java

```diff
@@ -27,12 +26,16 @@ public OutputPrinter(Logger logger) {
     }
 
     public void info(String message, Object... args) {
-        logger.info(message, args);
-        System.out.println(MessageFormatter.arrayFormat(message, args).getMessage());
+        final String errorMessage = MessageFormatter.arrayFormat(message, args).getMessage();
```

Comment:
We don't really use final on variables anywhere as the Java compiler will automatically infer it now.